deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -206,8 +206,7 @@ spec:
                     more information, see https://www.graalvm.org/docs/reference-manual/aot-compilation/.
                   type: boolean
                 resources:
-                  description: Resources for build pods. Default limits are 1GB RAM/0.5
-                    CPU on JVM and 4GB RAM/1 CPU for native builds.
+                  description: Resources for S2I build pods.
```

Comment:
builder pods?

Reply:
Please do not change the CRDs directly!!! They are auto-generated and updated with the `operator-sdk` command line. 

Please update the `_types.go` file and then run `make vet` to apply the changes.